content/Mobile_Dev/WebAuthn/WebAuthn_Primer.adoc

```diff
@@ -0,0 +1,54 @@
+
+==== Hardware backed vs copyable credentials
+Some technology providers have begun to implement copyable credentials within their operating systems/ecosystems, oftentimes this is branded as passkeys. You should be aware of the difference between the two credential types so that you can guide your authentication strategy.
+
+Hardware backed credentials are credentials where the private key never leaves the piece of hardware that they were originally created on. This greatly minimizes the risk of your credential being exposed to attackers as they would need the physical device in order to access any of your accounts. These credentials allow for the highest degree of security.
+
+Copayable credentials are credentials that can be leveraged from one device to another. These credentials typically reside in a shared/cloud environment where an authenticated user is able to access all of their credentials from a single location. A user may also be able to pass their credentials to other users for shared access. While these credentials offer a high degree of usability, there is often a huge trade off in the degree of security offered.
+
+=== How do I deploy a WebAuthn relying party?
+The first major step in implementing and deploying a WebAuthn enabled application is to create a WebAuthn relying party. The relying party is the server-side component of your application, which contains the FIDO2 app. This FIDO2 app will be used to store and manage user credentials, as well as validating the registration and authentication ceremonies. The FIDO2 app typically requires an interface to your user store (identity provider) in order to help the authentication/registration processes. 
+
+For this step you will have two options. Your first option will walk you through how to stand up a local instance of a WebAuthn server for you to test with a simple application. Your second option will deploy a full reference architecture using Yubico’s recommended best practices that includes both a relying party and full client application.
+
+* link:https://developers.yubico.com/WebAuthn/WebAuthn_Walk-Through.html[Yubico java-webauthn-server demo] - Walkthrough on how to deploy a local instance of a WebAuthn server
+* link:https://developers.yubico.com/Developer_Program/WebAuthn_Starter_Kit/[Yubico WebAuthn Starter Kit] - Full reference architecture with both a relying party and client application. More information will be given in the next section
+
+=== How do I deploy an example WebAuthn application?
+The next major component needed for a WebAuthn application is the client application. This is what your end user will see when they attempt to use your application. The client should facilitate the authentication, registration, and account management processes. In order to enable this in your specific application you will need a way to communicate with the relying party provisioned in the previous section.
```

Comment:
Consider adding a bulleted list of deployment considerations / learning objectives of the WebAuthn Starter Kit here. The offer the starter kit as the 'go here to dive deeper and learn more'

Reply:
Changed in the latest commit